doc/classes/Object.xml

```diff
@@ -96,9 +103,16 @@
 			</argument>
 			<description>
 				Calls the [code]method[/code] on the object and returns the result. This method supports a variable number of arguments, so parameters are passed as a comma separated list. Example:
-				[codeblock]
-				call("set", "position", Vector2(42.0, 0.0))
-				[/codeblock]
+				[codeblocks]
+				[gdscript]
+				var node = Node2D.new()
```

Comment:
Why add the instantiation of a `Node2D`? Code can be assumed to be run in a class that inherits `Node2D`, and the original example was thus just fine. This adds redundant code which is not necessary to understand the use of those methods.

Reply:
IMO it is not clear (it is clear when you look at what is called), because we are in the Object class. That was my reasoning

Reply:
Yeah I had meant to delete this comment actually but forgot too. I agree that since this is in Object, it can make sense to make it explicit.